chia/simulator/add_blocks_in_batches.py

```diff
@@ -35,20 +34,12 @@ async def add_blocks_in_batches(
 
     vs = ValidationState(ssi, diff, None)
 
-    for block_batch in to_batches(blocks, 64):
-        b = block_batch.entries[0]
-        if (b.height % 128) == 0:
-            print(f"main chain: {b.height:4} weight: {b.weight}")
-        # vs is updated by the call to add_block_batch()
-        success, state_change_summary = await full_node.add_block_batch(
-            block_batch.entries, PeerInfo("0.0.0.0", 0), fork_info, vs
-        )
-        assert success is True
-        if state_change_summary is not None:
-            peak_fb: Optional[FullBlock] = await full_node.blockchain.get_full_peak()
-            assert peak_fb is not None
-            ppp_result: PeakPostProcessingResult = await full_node.peak_post_processing(
-                peak_fb, state_change_summary, None
-            )
-            await full_node.peak_post_processing_2(peak_fb, None, state_change_summary, ppp_result)
+    # vs is updated by the call to add_block_batch()
+    success, state_change_summary = await full_node.add_block_batch(blocks, PeerInfo("0.0.0.0", 0), fork_info, vs)
```

Comment:
we call add_blocks_in_batches sometimes with huge lists of of blocks since we expect it to batch them into validation, this seems to diverge more then what we do in the full node

Reply:
can change the name to add_blocks but is this similar to what we currently do in the full node ?
in the full node we use a fixed batch size both for the short_sync_batch and sync_from_fork_point
the point of this method was to make the test code more similar to what we do in full_node.py so we dont hide all sorts of bugs/issues